mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -54,23 +66,140 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 9 underlineColor
 		ARG 10 light
 	METHOD method_1726 isRightToLeft ()Z
-	METHOD method_1727 getStringWidth (Ljava/lang/String;)I
-	METHOD method_1728 wrapStringToWidthAsList (Lnet/minecraft/class_2561;I)Ljava/util/List;
+	METHOD method_1727 getWidth (Ljava/lang/String;)I
+		ARG 1 text
+	METHOD method_1728 wrapLines (Lnet/minecraft/class_2561;I)Ljava/util/List;
+		ARG 1 text
 		ARG 2 width
 	METHOD method_1729 draw (Lnet/minecraft/class_4587;Ljava/lang/String;FFI)I
+		ARG 1 matrices
+		ARG 2 text
+		ARG 3 x
+		ARG 4 y
+		ARG 5 color
 	METHOD method_22941 draw (Lnet/minecraft/class_2561;FFILnet/minecraft/class_1159;Z)I
+		ARG 1 text
 		ARG 2 x
 		ARG 3 y
 		ARG 4 color
 		ARG 5 matrix
 		ARG 6 shadow
 	METHOD method_22942 draw (Lnet/minecraft/class_2561;FFIZLnet/minecraft/class_1159;Lnet/minecraft/class_4597;ZII)I
+		ARG 1 text
 		ARG 2 x
 		ARG 3 y
 		ARG 4 color
 		ARG 5 shadow
 		ARG 6 matrix
-		ARG 7 vertexConsumerProvider
+		ARG 7 vertexConsumers
+		ARG 8 seeThrough
+		ARG 9 backgroundColor
+		ARG 10 light
+	METHOD method_27515 tweakTransparency (I)I
```

Comment:
How does it tweak the transparency?

Reply:
it removes the transparent bits and blackshift the color if i remembered correctly

Reply:
Maybe add a JD comment explaining this?

Reply:
Private method, no need to